shanghai.example.yaml

```diff
@@ -4,6 +4,7 @@ networks:
     # serves as base for each network
     # networks can overwrite any setting here.
     nick: Shanghai
+    user: Shanghai
```

Comment:
That's what `ident` was meant to be (I think it was meant to be used together with the `identd` service of a user in very old and very original versions of the internet). But `user` is what's being used in rfc, so I guess we should completely remove `ident`.